packages/react-next/src/components/SpinButton/SpinButton.test.tsx

```diff
@@ -162,7 +163,7 @@ describe('SpinButton', () => {
     const inputDOM = wrapper.getDOMNode().querySelector('input')!;
     expect(inputDOM.value).toBe('');
     expect(inputDOM.getAttribute('aria-valuenow')).toBe(null);
-    expect(inputDOM.getAttribute('aria-valuetext')).toBe('');
+    expect(inputDOM.getAttribute('aria-valuetext')).toBe(null);
```

Comment:
I think this is actually correct but I wonder at what made you realize this needed to be changed?

Reply:
I updated the code and found a problem related to this and resolved it.